timon/plugins/http_server/views.py

```diff
@@ -64,11 +91,23 @@ async def get_heap_len():
     return data_to_return
 
 
+@app.route("/heap/probe/<probename>/")
+async def search_probe_in_heap(probename):
+    """
+    Search a probe in the heap, and returns it if exists else returns a 404
```

Comment:
returns it if it exists

or

returns it if existing

Reply:
done